### DISCUSSION TITLE
Summary doesn't update after editing in Mobile

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#125228] [Mobile] [answered]
GitHub Mobile beta for Android 

1. Open a ticket 
https://github.com/nektos/act/issues/2334#issuecomment-2125421982
2. Edit the summary
3. Save the changes
4. See the unedited title:
    ![Screenshot_20240522-140344.png](https://github.com/community/community/assets/2119212/f1b260b2-a6ba-4310-94f5-a4867537fbbe)

Expected results: change should be reflected without having to reload the issue. 

This wouldn't be a problem in a web browser.

**Comment by kangarruu** [comment, marked as answer]
Hi @jsoref thank you for reporting! We'll consider this improvement in future work 👍🏻
